### PR TITLE
[sec]: Initial security policy

### PR DESCRIPTION
**Pull request by evenstensberg**

### CODE REVIEW

**Comment by evenstensberg** [comment]
Needs to be fleshed out first

**Comment by ematipico** [comment]
What's the next work on this?

**Comment by evenstensberg** [comment]
We need to say that we do not support webpack < 4 etc and webpack-cli < v.3, then we need to give a text about how to report security issues

**Comment by evenstensberg** [comment]
ptal

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+This document explains the security policy of webpack-cli and how we intend to support webpack and webpack-cli.
+
+## Supported Versions
+
+webpack CLI is currently supporting webpack v4 and webpack v5. Security fixes are released in patches.
+
+| webpack version | webpack-cli version           | Supported          |
+| --------------- | ----------------------------- | ------------------ |
+| 4.x.0           | ^3.1.2                        | :white_check_mark: |
```
Comment (ematipico) [inline SECURITY.md]: We have to rewrite this bit. 
After webpack `4.20.0` => min version required is `3.1.1`


https://github.com/webpack/webpack/releases/tag/v4.20.0

Comment (evenstensberg) [reply SECURITY.md]: gotcha, added now

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+This document explains the security policy of webpack-cli and how we intend to support webpack and webpack-cli.
+
+## Supported Versions
+
+webpack CLI is currently supporting webpack v4 and webpack v5. Security fixes are released in patches.
+
+| webpack version | webpack-cli version           | Supported          |
+| --------------- | ----------------------------- | ------------------ |
+| > 4.20.0        | ^3.1.2                        | :white_check_mark: |
```
Comment (ematipico) [inline SECURITY.md]:
```suggestion
| >= 4.20.x        | ^3.1.2                        | :white_check_mark: |
```

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+| 4.20.x          | ^3.1.2                        | :white_check_mark: |
```
Comment (ematipico) [inline SECURITY.md]:
```suggestion
| < 4.19.x          | ^3.1.1                        | :white_check_mark: |
```

**Comment by evenstensberg** [comment]
Appeneded @ematipico

**Pull request merged**